client/jetpack-connect/jetpack-new-site/style.scss

```diff
@@ -143,18 +143,18 @@
 
 		&:hover,
 		&:focus {
-			border-color: darken( $green-jetpack, 30% );
+			border-color: var( --color-jetpack-dark );
 		}
 
 		&:focus {
-			box-shadow: 0 0 0 2px lighten( $green-jetpack, 25% );
+			box-shadow: 0 0 0 2px var( --color-jetpack-light );
 		}
 
 		&[disabled],
 		&:disabled {
 			background: tint( $green-jetpack, 50% );
 			border-color: tint( $green-jetpack, 35% );
```

Comment:
I'm not feeling too confident on if variables work here. If they don't, I'm a bit stumped on which to use for there to be a visible difference.

Reply:
We recently aligned disabled buttons to look the same across button types. I think that should be the same here as well. Compare with #30229.